auth-web/src/routes/router.ts

```diff
@@ -120,35 +121,70 @@ export function getRoutes (): RouteConfig[] {
     { path: '/', name: 'root', redirect: 'home' },
     {
       path: '/home',
-      name: 'home',
       component: HomeView,
       children: [
         {
           path: '',
+          name: 'home',
           redirect: 'decide-business'
         },
         {
           path: 'decide-business',
+          name: 'decide-business',
           component: DecideBusinessView,
-          meta: { showNavBar: false }
+          meta: {
+            breadcrumb: [
+              {
+                text: HomeBreadCrumb.text,
+                to: { name: 'decide-business' }
+              }
+            ],
+            showNavBar: true
+          }
         },
         {
           path: 'request-name',
+          name: 'request-name',
           component: RequestNameView,
-          meta: { showNavBar: false }
+          meta: {
+            breadcrumb: [
+              {
+                text: HomeBreadCrumb.text,
+                to: { name: 'request-name' }
+              }
+            ],
+            showNavBar: true
+          }
         },
         {
           path: 'incorporate-or-register',
+          name: 'incorporate-or-register',
           component: IncorpOrRegisterView,
-          meta: { showNavBar: false }
+          meta: {
+            breadcrumb: [
```

Comment:
will it be better to keep breadcrumb in one place? either in resources file or inline like this?

Reply:
It can be done like that when the crumb is common but each of these routes require a unique `to` defined. 
Not sure exactly what you mean. 
Do you want to define all of these in a resource and import then into the routes?

Reply:
I think in these cases it's ok in the routes but if we were to re use these specific crumbs it would make sense to make it a resource. I just can't think of where else we would reuse at this time.

Reply:
In someplace, used as breadcrumb: [DashboardBreadcrumb], and other places using only text from resources (text: HomeBreadCrumb.text). So I was thinking to use one pattern. or even if we are using only text, better to add in the language file and use it in both places. (ie in resource text  and HomeBreadCrumb.text). I think its fine now as you mentioned above :)

Reply:
Ahh i see you what you mean, absolutely that makes sense.